pulsar-client-api/src/main/java/org/apache/pulsar/client/api/Consumer.java

```diff
@@ -523,11 +523,17 @@ CompletableFuture<Void> reconsumeLaterCumulativeAsync(Message<?> message,
 
     /**
      * The asynchronous version of {@link Consumer#seek(MessageId)}.
+     * <p>
+     * If there is already a seek operation in progress, the method will log a warning and 
+     * return a future completed exceptionally.
```

Comment:
It's probably worth documenting on the `seek` methods too. If a user were to share a consumer's reference in two threads, it could lead to the same error.